### ISSUE TITLE
Cannot hide Id field

### ISSUE THREAD

**Issue by Intelligent2013** [la-team/light-admin#244]
How can I hide the 'id' field? I think this field is intended for internal database structure purposes and user should not has the possibility to edit this field. Another solution is set the 'id' field to read-only.

**Comment by elwin013** [comment]
Field id cannot be set by user (it is read only) and it's only viewed for information purposes. Do you really need to hide id field?

**Comment by pnarem** [comment]
Yes, the users of the application always wonder about the significance and
asking us to hide it, if there is no business purpose.

**Comment by pnarem** [comment]
And also how do you connect to multiple datasource with the light admin
application? can anybody help with an example?

**Comment by adeptues** [comment]
Having problems with this, as the id is editable on the form view when creating a new entity, as its default set to 0 it overrides what ever is set with that id in the db which means i need to manually change the id to something not already in the db when creating a new entity
